tests/test_simple.py

```diff
@@ -0,0 +1,5 @@
+from subprocess import check_output
+import pytest
+
+def test_simple():
+    check_output('python find_unicode_control2.py *', shell=True)
```

Comment:
This should be `find_unicode_control.py` now I guess?

Reply:
rebased